### PR TITLE
Fix(eos_cli_config_gen,eos_designs): Accept numeric ACL names

### PR DESCRIPTION
**Pull request by pmprado**
## Related Issue(s)

No issue opened for this

## Component(s) name

`arista.avd.eos_cli_config_gen`

### User Checklist

- N/A

### Repository Checklist

- [x] My code has been rebased from devel before I start
- [x] I have read the [**CONTRIBUTING**](https://avd.sh/en/latest/docs/contribution/overview.html) document.
- [x] My change requires a change to the documentation and documentation have been updated accordingly.
- [X] I have updated [molecule CI](https://github.com/aristanetworks/ansible-avd/tree/devel/ansible_collections/arista/avd/molecule) testing accordingly. (check the box if not applicable)

### CODE REVIEW

File: ansible_collections/arista/avd/molecule/eos_cli_config_gen/inventory/host_vars/acl.yml [hunk 1]
```diff
@@ -61,3 +70,14 @@ standard_access_lists:
...
+    counters_per_entry: false
+    sequence_numbers:
+      - sequence: 10
+        action: "remark ACL to restrict access RFC1918 addresses"
+      - sequence: 20
+        action: "permit 10.0.0.0/8"
+      - sequence: 30
+        action: "permit 172.16.0.0/12"
+      - sequence: 40
+        action: "permit 192.168.0.0/16"
```
Comment (ClausHolbechArista) [inline ansible_collections/arista/avd/molecule/eos_cli_config_gen/inventory/host_vars/acl.yml]: missing a newline at the end.

File: ansible_collections/arista/avd/roles/eos_cli_config_gen/schemas/eos_cli_config_gen.schema.yml [hunk 2]
```diff
@@ -328,6 +328,7 @@ keys:
         name:
           type: str
           description: Access-list Name
+          convert_types: [ int ]
```
Comment (ClausHolbechArista) [inline ansible_collections/arista/avd/roles/eos_cli_config_gen/schemas/eos_cli_config_gen.schema.yml]: You should not change this file manually. You need to change the fragments which will then be joined (by pre-commit) to this file.

**Comment by ClausHolbechArista** [comment]
Moving to draft until tests are passing.

File: ansible_collections/arista/avd/molecule/eos_cli_config_gen/inventory/host_vars/acl.yml (hunk 1 above)
Comment (pmprado) [reply ansible_collections/arista/avd/molecule/eos_cli_config_gen/inventory/host_vars/acl.yml]: Ack, added

File: ansible_collections/arista/avd/roles/eos_cli_config_gen/schemas/eos_cli_config_gen.schema.yml (hunk 2 above)
Comment (pmprado) [reply ansible_collections/arista/avd/roles/eos_cli_config_gen/schemas/eos_cli_config_gen.schema.yml]: Ack - moved the changes to the fragments

**Review by carlbuchmann — APPROVED** [review]
LGTM

**Review by ClausHolbechArista — APPROVED** [review]

**Pull request merged**